lib/src/data/datasource/remote/recipes_api_service.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:dio/dio.dart';
+import 'package:get/get_connect/http/src/status/http_status.dart';
+import 'package:spoonacular/src/core/resource/data_state.dart';
+import 'package:spoonacular/src/data/model/recipes_list_model.dart';
+import 'package:spoonacular/src/domain/entity/recipes_list_entity.dart';
+
+class RecipesApiService {
+  RecipesApiService();
+
+  static const String uri = 'https://api.spoonacular.com/';
+  static const String apiKey = '6add5cb5264c4077a6540c34ca241c8b';
+  static const String uriDio = '${uri}recipes/random?apiKey=$apiKey&number=3';
```

Comment:
You can use the apiKey as a query parameter, but you can also use it like this
`dio.get("endpoint", options: dio.Options(headers: headers))`
headers is a Map, inside this map you can set you apiKey

Reply:
Done